elf/module.go

```diff
@@ -66,6 +66,19 @@ int bpf_prog_attach(int prog_fd, int target_fd, enum bpf_attach_type type)
 	// BPF_PROG_ATTACH = 8
 	return syscall(__NR_bpf, 8, &attr, sizeof(attr));
 }
+
+int bpf_prog_detach(int prog_fd, int target_fd, enum bpf_attach_type type)
+{
+	union bpf_attr attr;
+
+	bzero(&attr, sizeof(attr));
+	attr.target_fd	   = target_fd;
+	attr.attach_bpf_fd = prog_fd;
+	attr.attach_type   = type;
+
+	// BPF_PROG_DETACH = 9
+	return syscall(__NR_bpf, 9, &attr, sizeof(attr));
```

Comment:
We have `BPF_PROG_ATTACH` / `BPF_PROG_DETACH` defined in `elf/include/bpf.h` - any reason we don't use them?